### ISSUE TITLE
weird layout issue

### ISSUE THREAD

**Issue by danipralea** [superpeteblaze/ScalingCarousel#34]
Hello! 
Would you know what causes this? It's appearing each time I call `reloadData()` on the carousel. The carousel is in a table view cell.
<img width="398" alt="screenshot 2018-03-28 19 53 22" src="https://user-images.githubusercontent.com/768893/38044213-ce7739de-32c1-11e8-8ec3-3e0d1b8043cd.png">

Only if I start to scroll a bit it goes back to how it should appear:
<img width="399" alt="screenshot 2018-03-28 19 55 08" src="https://user-images.githubusercontent.com/768893/38044271-eece993e-32c1-11e8-8ee8-f397cf665b5e.png">

**Comment by emorvillez** [comment]
Hi, I found same issue. 
To fix it when I load fresh data in my ScalingCarouselCell, I call self.layoutSubviews() so when you call 

`class MyViewCell: ScalingCarouselCell {

    var data: String? {
        didSet {
            if let data = data {
                //...
                self.layoutSubviews()
            }
        }
    }`

**Comment by danipralea** [comment]
sweet. that worked for me as well. thanks a lot @emorvillez!

**Comment by superpeteblaze** [comment]
Not sure at the moment @danipralea, but I'll create an issue and look in to it. Thanks for reporting 👍🏼. Nice workaround @emorvillez 🎉

**Comment by danipralea** [comment]
Thanks @superpeteblaze. I will try to add it to the tableHeaderView property. maybe it has something to do with the reloading of the table. will keep you updated.

it's funny that I have the same carousel in other 2 tab screens and those are laying out ok. same setup. in a table, in a cell.

**Comment by superpeteblaze** [comment]
Cool, thanks

**Comment by superpeteblaze** [comment]
HI @danipralea, the only solution I found was to force a `layoutSubviews` call on the cell, similar to what @emorvillez mentioned. However, it is not advisable to call `layoutSubviews` directly, so I recommend adding the following lines after you configure your cell for display(e.g in`cellForItem(at:)`):

```
cell.setNeedsLayout()
cell.layoutIfNeeded()
```

I have updated the readMe to specify this step, and will now close the issue.

Thanks again for the feedback and help guys! 👍🏼
